deploy/google-cas-issuer.yaml

```diff
@@ -462,7 +412,7 @@ spec:
         - --zap-devel=true
         command:
         - /google-cas-issuer
-        image: quay.io/jetstack/google-cas-issuer:latest
+        image: quay.io/jetstack/cert-manager-google-cas-issuer:latest
```

Comment:
I don't see any related changes to the build scripts in this PR. Was the Docker image name changed somewhere else?

Reply:
This was a mistake accidentally included from another branch where I am writing an e2e test suite